### PR TITLE
feat: framer added

### PR DESCRIPTION
**Pull request by Lalman888**
framer added #133

### CODE REVIEW

**Comment by Lalman888** [comment]
@gabrysia694  any suggestions you want to give related to animation ?

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -28,14 +37,18 @@ export default function HomePage() {
...
         </div>
       </section>
       <section>
         <div className="mx-auto max-w-screen-2xl px-4 py-8 sm:px-6 lg:px-8">
           <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
-            <div className="bg-blue-900 p-8 md:p-12 lg:px-16 lg:py-24">
+            <motion.div 
+            // initial={{ opacity: 0, y: 200 }}
+            // animate={{ opacity: 1, y: 0 }}
+            // transition={{ duration: 2 }}
```
Comment (gabrysia694) [inline src/pages/HomePage.jsx:51]: @Lalman888 did you want to add an animation here as well?

Comment (Lalman888) [reply src/pages/HomePage.jsx:51]: I wanted to add animation, but don't look good any suggestions ?

Comment (gabrysia694) [reply src/pages/HomePage.jsx:51]: maybe a scroll-triggered animation that starts when the element is shown? could defining only opacity

Comment (gabrysia694) [reply src/pages/HomePage.jsx:51]: and `overflow-x-hidden` could be added
dd

Comment (Lalman888) [reply src/pages/HomePage.jsx:51]: okay I will do that

Comment (Lalman888) [reply src/pages/HomePage.jsx:51]: here https://deploy-preview-146--gymjunkies.netlify.app/        I didn't see any problem 
 
image

Comment (gabrysia694) [reply src/pages/HomePage.jsx:51]: I meant the horizontal scrollbar is showing up for the time of animation and we could hide it so it looks better

Comment (Lalman888) [reply src/pages/HomePage.jsx:51]: Can you merge this then again I will open new pull request for overflow-x-hidden this creating merge conflict .

**Review by gabrysia694 — APPROVED** [review]

File: src/pages/HomePage.jsx (hunk 1 above)
Comment (gabrysia694) [reply src/pages/HomePage.jsx:51]: done

**Pull request merged**